facebook/OpenBIC
# common: shell: Modify shell sensor command

Summary:
- Shell sensor read command support real value display if using 4-byte accuracy reading.
- Shell sensor read add polling time display.

Dependency: #461 

Test Plan:
- Build Code: PASS
- Check result from Waimea Canyon BIC console: PASS

Log:
- BIC console:
  ```
  uart:~$ platform sensor list_all
  ---------------------------------------------------------------------------------
  [0x1 ] MB Inlet Temp            : tmp75      | access[O] | 4byte_acur_read_success   | 1    sec | 38.00
  [0x2 ] MB Outlet Temp           : tmp75      | access[O] | 4byte_acur_read_success   | 1    sec | 30.00
  [0x3 ] IOM Temp                 : tmp75      | access[O] | 4byte_acur_read_success   | 1    sec | 31.00
  [0x8 ] SSD0 Temp                : nvme       | access[O] | 4byte_acur_read_success   | 1    sec | 32.00
  [0x5 ] SOC CPU Temp             : peci       | access[O] | 4byte_acur_read_success   | 1    sec | 98.00
  [0x6 ] SOC Therm Margin         : peci       | access[O] | 4byte_acur_read_success   | 1    sec | -2.00
  [0x7 ] CPU TjMax                : peci       | access[O] | 4byte_acur_read_success   | 1    sec | 100.00
  [0xa ] DIMMA Temp               : peci       | access[O] | 4byte_acur_read_success   | 1    sec | 31.00
  [0xb ] DIMMC Temp               : peci       | access[O] | 4byte_acur_read_success   | 1    sec | 30.00
  [0xc ] DIMME Temp               : peci       | access[O] | 4byte_acur_read_success   | 1    sec | 33.00
  [0xd ] DIMMG Temp               : peci       | access[O] | 4byte_acur_read_success   | 1    sec | 33.00
  [0x60] CPU Pwr                  : peci       | access[O] | 4byte_acur_read_success   | 1    sec | 96.00
  [0x20] P12V_STBY Vol            : adc        | access[O] | 4byte_acur_read_success   | 1    sec | 11.71
  [0x26] P12V_DIMM Vol            : adc        | access[O] | 4byte_acur_read_success   | 1    sec | 11.73
  [0x22] P3V3_STBY Vol            : adc        | access[O] | 4byte_acur_read_success   | 1    sec | 3.28
  [0x24] P1V05_PCH Vol            : adc        | access[O] | 4byte_acur_read_success   | 1    sec | 1.00
  [0x21] P3V_BAT Vol              : adc        | access[O] | 4byte_acur_read_success   | 1    sec | 0.77
  [0x28] P3V3_M2 Vol              : adc        | access[O] | 4byte_acur_read_success   | 1    sec | 3.27
  [0x25] P5V_STBY Vol             : adc        | access[O] | 4byte_acur_read_success   | 1    sec | 4.77
  [0x23] P1V8_STBY Vol            : adc        | access[O] | 4byte_acur_read_success   | 1    sec | 1.70
  [0x27] P1V2_STBY Vol            : adc        | access[O] | 4byte_acur_read_success   | 1    sec | 1.19
  [0x2d] VCCD VR Vol              : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 1.14
  [0x2e] FAON VR Vol              : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 0.97
  [0x2c] EHV VR Vol               : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 1.80
  [0x2a] VCCIN VR Vol             : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 1.76
  [0x2b] FIVRA VR Vol             : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 1.80
  [0x44] VCCD VR Cur              : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 2.00
  [0x45] VCCD VR Cur in           : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 0.23
  [0x46] FAON VR Cur              : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 9.70
  [0x43] EHV VR Cur               : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 0.90
  [0x41] VCCIN VR Cur             : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 43.60
  [0x42] FIVRA VR Cur             : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 3.90
  [0x11] VCCD SPS Temp            : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 40.00
  [0x12] FAON SPS Temp            : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 43.00
  [0x10] EHV SPS Temp             : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 38.00
  [0xe ] VCCIN SPS Temp           : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 50.00
  [0xf ] FIVRA SPS Temp           : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 49.00
  [0x69] VCCD VR Pout             : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 2.00
  [0x6a] FAON VR Pout             : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 9.00
  [0x68] EHV VR Pout              : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 1.00
  [0x66] VCCIN VR Pout            : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 76.00
  [0x67] FIVRA VR Pout            : isl69259   | access[O] | 4byte_acur_read_success   | 1    sec | 8.00
  [0x4 ] PCH Temp                 : pch        | access[O] | 4byte_acur_read_success   | 1    sec | 45.00
  [0x62] DIMMA PMIC_Pout          : pmic       | access[O] | 4byte_acur_read_success   | 1    sec | 0.00
  [0x63] DIMMC PMIC_Pout          : pmic       | access[O] | 4byte_acur_read_success   | 1    sec | 0.00
  [0x64] DIMME PMIC_Pout          : pmic       | access[O] | 4byte_acur_read_success   | 1    sec | 0.12
  [0x65] DIMMG PMIC_Pout          : pmic       | access[O] | 4byte_acur_read_success   | 1    sec | 0.00
  [0x6b] IOM INA Pwr              : ina230     | access[O] | 4byte_acur_read_success   | 1    sec | 1.74
  [0x47] IOM INA Cur              : ina230     | access[O] | 4byte_acur_read_success   | 1    sec | 0.14
  [0x2f] IOM INA vol              : ina230     | access[O] | 4byte_acur_read_success   | 1    sec | 11.98
  [0x9 ] HSC Temp                 : mp5990     | access[O] | 4byte_acur_read_success   | 1    sec | 39.00
  [0x29] HSC Input Vol            : mp5990     | access[O] | 4byte_acur_read_success   | 1    sec | 11.94
  [0x40] HSC Output Cur           : mp5990     | access[O] | 4byte_acur_read_success   | 1    sec | 4.25
  [0x61] HSC Input Pwr            : mp5990     | access[O] | 4byte_acur_read_success   | 1    sec | 48.00
  ---------------------------------------------------------------------------------
  ```
